### DISCUSSION TITLE
Ability to filter based on Bitrise jobs that I started

### DISCUSSION THREAD

**Discussion by tinder-maxwellelliott** [deszip/BitBot#19] [Feature requests]
Hello There,

Thanks for the wonderful tool. I am looking into adding the ability to filter jobs to only show ones that I have started. Where should I first look to add such code to the app? Thanks

**Comment by deszip** [comment]
Hi @tinder-maxwellelliott , thanks for suggestion and willing to help :) Looks like a useful feature.

Some thoughts:
- you need to add filtering to `BRAppsDataSource` to alter predicate, for now it's showing all builds and you need some API to show only those matching your filter. Internally it will add predicate to fetch request or remove it when you switch filtering off.
- talking about filtering your builds, it's probably `triggeredBy` field on `BRBuild` but I'm not sure.
- you need some UI for that, you can make a PR with functionality only and I'll ask designer to make a UI for that feature
